docs/source/markdown/podman.1.md

```diff
@@ -336,9 +336,11 @@ Images are pulled under `XDG_DATA_HOME` when specified, otherwise in the home di
 
 Currently the slirp4netns package is required to be installed to create a network device, otherwise rootless containers need to run in the network namespace of the host.
 
+In certain enviroments like HPC (High Performance Computing) users are not allowed to take advantage of the additional UIDs and GIDs from the /etc/subuid and /etc/subgid systems.  Even in this environment you are allowed to use rootless podman with a single UID.  You need to set the `ignore_chown_errors` option in the /etc/containers/storage.conf or in ~/.config/containers/storage.conf. This option tells podman when pulling an image to ignore chown errors when it attempts to change a file in a container image to match the non root UID in the image. This means all files get saved as the user's UID. Note this could cause issues when running the container.
+
 ### **NOTE:** Unsupported file systems in rootless mode
 
-The Overlay file system (OverlayFS) is not supported in rootless mode.  The fuse-overlayfs package is a tool that provides the functionality of OverlayFS in user namespace that allows mounting file systems in rootless environments.  It is recommended to install the fuse-overlayfs package.  In rootless mode Podman will automatically use the fuse-overlafs program as the mount_program if installed, as long as the $HOME/.config/containers/storage.conf file was not previously created.  If storage.conf exists in the homedir, add `mount_program = "/usr/bin/fuse-overlayfs"` under `[storage.options.overlay]` to enable this feature.
+The Overlay file system (OverlayFS) is not supported with older kernels in rootless mode.  The fuse-overlayfs package is a tool that provides the functionality of OverlayFS in user namespace that allows mounting file systems in rootless environments.  It is recommended to install the fuse-overlayfs package.  In rootless mode Podman will automatically use the fuse-overlafs program as the mount_program if installed, as long as the $HOME/.config/containers/storage.conf file was not previously created.  If storage.conf exists in the homedir, add `mount_program = "/usr/bin/fuse-overlayfs"` under `[storage.options.overlay]` to enable this feature.
```

Comment:
Any time I see "older", "newer", "recent", I ask myself: how will this read in six/twelve months? Would it be safe to say here, "with kernels prior to 5.12.9"?

Reply:
No because it will be backported into RHEl8.

Reply:
How about if we just say `is not supported in all kernels ...`